elasticdl/python/ps/optimizer_wrapper.py

```diff
@@ -307,12 +324,6 @@ def _delete_variables(self):
                         break
                     else:
                         opt_weight_iter += 1
-        for key in list(self._tls._slot_variables.keys()):
-            del self._tls._slot_variables[key]
-
-        # Delete variables in embed_variables.
-        for key in list(self._tls._embed_variables.keys()):
-            del self._tls._embed_variables[key]
```

Comment:
Rename method name, and update comments.

By the way, all delete those comments related to Redis in this file.